Makefile

```diff
@@ -213,6 +215,8 @@ test/e2e/multitenant-rhoam/prow: export INSTALLATION_PREFIX := sandbox-rhoam
 test/e2e/multitenant-rhoam/prow: export IDENTITY_PROVIDER_NAME ?= DevSandbox
 test/e2e/multitenant-rhoam/prow: export INSTALLATION_NAME := rhoam
 test/e2e/multitenant-rhoam/prow: export INSTALLATION_SHORTHAND := rhoam
```

Comment:
hey @makslion i think this needs to be updated to match whats in this PR https://github.com/integr8ly/integreatly-operator/pull/2125 so that when in prow we are also using this new config.

Reply:
Also, from multitenant logs it seems we are getting `Error: invalid format of image, use one of the following options:`. I dont think updating the INSTALLATION SHORTHAND will adress this issue but maybe its worth logging the value of IMAGE_FORMAT just to see why its failing to deploy operator?